data/json/npcs/godco/NPC_Tom_Powell.json

```diff
@@ -0,0 +1,204 @@
+        "topic": "TALK_GODCO_Tom_Firstmeet"
+      },
+      {
+        "text": "Hi Tom, got a minute?",
+        "condition": { "npc_has_var": "u_met_godco_tom", "type": "general", "context": "meeting", "value": "yes" },
+        "topic": "TALK_GODCO_Tom_2"
+      },
+      { "text": "Hey, Tom.  I can't stay to talk.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Tom_Firstmeet",
+    "dynamic_line": "Was it difficult to find us?  We haven't seen a single soul out there for so long, I'm starting to lose my hope.  So, tell me about yourself.",
+    "responses": [
+      {
+        "text": "I'm just a curious traveler.  What about you, how are you doing?",
+        "condition": { "not": { "u_has_var": "godco_joinee", "type": "general", "context": "meeting", "value": "yes" } },
+        "topic": "TALK_GODCO_Tom_Firstmeet2"
+      },
+      {
+        "text": "I'm seeking asylum here.  What about you, how are you doing?",
+        "condition": { "u_has_var": "godco_joinee", "type": "general", "context": "meeting", "value": "yes" },
+        "topic": "TALK_GODCO_Tom_Firstmeet2"
+      },
+      { "text": "Actually I'm just heading out.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Tom_Firstmeet2",
+    "dynamic_line": "Hey, that's nice of you to ask.  The people here are barely holding on.  Malnutrition, sleepless nights, odd sounds from the woods, everyone turns a little mad in these circumstances.  It's not their fault, I know, but they did get what they wanted.",
+    "responses": [
+      { "text": "Please continue.", "topic": "TALK_GODCO_Tom_Firstmeet3" },
+      { "text": "No rest for the weary.  See you around, Tom.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Tom_Firstmeet3",
+    "dynamic_line": "The sky has broken!  They received a rather passive-aggressive sign from God!  The wait is finally over.  Well, not for me.  So much is left unanswered, and I know for a fact the answer's out there.",
+    "responses": [
+      { "text": "Perhaps it's not worth finding out.", "topic": "TALK_GODCO_Tom_Firstmeet4" },
+      { "text": "Try not to get stuck in the past.  See you around, Tom.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Tom_Firstmeet4",
+    "dynamic_line": "Perhaps.  You don't wanna think about the stuff I'm thinking about, it would spiral a newbie independent thinker into madness.  I don't blame you for saying that.",
+    "responses": [ { "text": "I'd better get going.", "topic": "TALK_DONE" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Tom_2",
+    "dynamic_line": "I'm currently in the middle of solving a pointless mystery for my own entertainment.  Of course I have time!",
+    "responses": [ { "text": "Sorry Tom, I can't stay to chat.", "topic": "TALK_DONE" } ]
```

Comment:
You just asked if he has a minute and when he says yes the only response available is "Sorry I can't talk now"

Reply:
The rest of the topics are in a cover-all response list towards the top. This helps to cut down on repetitive definitions for the same thing across the chat.